### ISSUE TITLE
Donate command extensions - Flavor Text and /History

### ISSUE THREAD

**Issue by Sritterbush** [Arx-Game/arxcode#112]
Donate is often used to reflect an IC purpose, and staff have encouraged doing so in some cases.

Proposal: Give donate a space for a TLDR length reason for the donation being applied. Example: To free child thralls in Grimhall.

Second Proposal: Add a donate/history so that a person can review the reasons given for their past uses of the donate command.

Result:  The reason people have donated to various organizations are now visible to staff and can be used to bolster story, plus people who are forgetful (like me) can just go check why we donated some silver someplace and go _ooooooh riiiiiiiight._

**Comment by TehomCD** [comment]
Yeah, that sounds fine to me. Could add a text field to the donation model.
